docs/conf.py

```diff
@@ -55,15 +55,15 @@
 
 # General information about the project.
 project = 'NI Modular Instruments Python API'
-copyright = '2017-2018, National Instruments'
+copyright = '2017-2019, National Instruments'
 author = 'National Instruments'
 
 # The version info for the project you're documenting, acts as replacement for
 # |version| and |release|, also used in various other places throughout the
 # built documents.
 #
 # The full version, including alpha/beta/rc tags.
-release = '1.1.1.dev0'
+release = '0.2.1.dev0'
```

Comment:
Why is this being downgraded?

Reply:
Because we only have one docs/conf.py and the version will be the last driver to build. nise hasn't had a 1.0 yet.